expression/builtin_math_vec.go

```diff
@@ -20,7 +20,7 @@ import (
 	"math"
 	"strconv"
 
-	utilMath "github.com/pingcap/tidb/util/math"
+	utilMath "github.com/pingcap/tidb/util/mathutil"
 
 	"github.com/cznic/mathutil"
```

Comment:
Please replace the usage of github.com/cznic/mathutil with github.com/pingcap/tidb/util/mathutil and remove the utilMath alias.